packages/strapi-plugin-content-manager/admin/src/containers/SettingViewModel/ListLayout.js

```diff
@@ -82,6 +82,7 @@ function ListLayout({
 
       moveListField(index, atIndex);
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can be removed if unused ?

Reply:
No I'd rather keep it for the moment, we need to be really of what deps we put in a hook.